lib/motor.js

```diff
@@ -727,12 +729,28 @@ Motor.prototype.release = function() {
 
 Motor.prototype.resume = function() {
   var speed = this.speed();
-  this.dir(speed, this.direction);
+  this.dir(this.direction);
   this.start(speed);
 
   return this;
 };
 
+Motor.prototype.enable = function() {
+  var state = priv.get(this);
+  if (this.pins.enable && !this.enabled) {
```

Comment:
I guess this would need to change as well: `if (typeof this.pins.enable !== "undefined"...`